src/cashier/withdraw.es6

```diff
@@ -7,12 +7,11 @@ import liveapi from 'websockets/binary_websockets';
 import windows from 'windows/windows';
 import rv from 'common/rivetsExtra';
 import currencyDialog from 'cashier/currency';
-import _ from 'lodash';
+import {debounce} from 'lodash'
```

Comment:
@4p00rv I read an article long time back on this. I am not sure what and where you are referring which says that this is the right way. But the truth is, its not the right way. It will make the file bulky. You can check the lodash code. The way to test it out is to check the generated file size with two approaches.

I could not find the original article where I read, but a quick google got me this.
http://stackoverflow.com/questions/35250500/correct-way-to-import-lodash

Read it. It will be useful. If you ever get into React/Redux development, its kind of *mandatory* to use `import debounce from 'lodash/debounce'` coding, considered under best practices.